tea.go

```diff
@@ -115,8 +115,8 @@ type Program struct {
 	altScreenWasActive bool
 	ignoreSignals      bool
 
-	// Stores the original reference to stdin for cases where input is not a
-	// TTY on windows and we've automatically opened CONIN$ to receive input.
+	// Stores the original reference to stdin on Windows. If the input is not
+	// a TTY, we automatically open CONIN$ to receive input.
```

Comment:
I don't think a comma would have been enough to help me. I tried splitting the sentence differently to split up what it's for and when it's used.

I believe my initial confusion was because `where input is not a TTY` sounds like a general statement, so I thought the Windows command was just an example.